eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -796,14 +796,125 @@ votingSpec =
           logRatificationChecks addCCGaid
           -- Add to the rewards of the delegator to this SPO
           -- to barely make the threshold (51 %! 100)
-          registerAndRetirePoolToMakeReward $ delegatorCStaking1
+          registerAndRetirePoolToMakeReward delegatorCStaking1
           passEpoch
           lookupReward delegatorCStaking1 `shouldReturn` Coin 1_200_000
           -- The same vote should now successfully ratify the proposal
           -- NOTE: It takes 2 epochs for SPO votes as opposed to 1 epoch
           -- for DRep votes to ratify a proposal.
           passNEpochs 2
           getLastEnactedCommittee `shouldReturn` SJust (GovPurposeId addCCGaid)
+        describe "Proposal deposits contribute to active voting stake" $ do
```

Comment:
I think there should also be tests for delegation of deposit to `DRep`s

Reply:
Yes that is covered in a previous PR. The tests are present in the same module under the DReps hierarchy.